codecept.conf.js

```diff
@@ -6,14 +6,16 @@ const _ = require('lodash');
 let localConfig = {};
 
 try {
-  localConfig = require(path.resolve(__dirname, '../../codecept.conf'));
+  localConfig = require(path.resolve(process.cwd(), './codecept.conf'));
 } catch (err) {
   // eslint-disable-next-line no-console
   console.info('Local config not found, using defaults');
 }
 
+const tests = path.relative(__dirname, path.resolve(process.cwd(), './apps/**/features/**/*.js'));
```

Comment:
I know this is a bit nasty, but codecept can't handle being given absolute paths (issue imminent) and so we have to engineer a relative path.